mu/locale/es/LC_MESSAGES/mu.po

```diff
@@ -5,756 +5,1091 @@
+msgid "Replace all?"
+msgstr "¿Reemplazar todo?"
+
+#: mu/interface/editor.py:222 mu/logic.py:960
 msgid "untitled"
-msgstr "UNTITLED"
+msgstr "intitulado"
+
+#: mu/interface/main.py:68
+msgid "Mode"
+msgstr "Modo"
 
-#: mu/interface.py:590
+#: mu/interface/main.py:69
+msgid "Change Mu's mode of behaviour."
+msgstr "Cambia el modo de comportamiento de Mu."
+
+#: mu/interface/main.py:71
 msgid "New"
-msgstr "NEW"
+msgstr "Nuevo"
 
-#: mu/interface.py:591
+#: mu/interface/main.py:72
 msgid "Create a new Python script."
-msgstr "CREATE NEW PYTHON SCRIPT"
+msgstr "Crea un nuevo script de Python."
 
-#: mu/interface.py:592
+#: mu/interface/main.py:73
 msgid "Load"
-msgstr "LOAD"
+msgstr "Cargar"
 
-#: mu/interface.py:593
+#: mu/interface/main.py:74
 msgid "Load a Python script."
-msgstr "LOAD A PYTHON SCRIPT"
+msgstr "Cargar un script de Python."
 
-#: mu/interface.py:594
+#: mu/interface/main.py:75
 msgid "Save"
-msgstr "SAVE"
+msgstr "Guardar"
 
-#: mu/interface.py:595
+#: mu/interface/main.py:76
 msgid "Save the current Python script."
-msgstr "SAVE THE CURRENT PYTHON SCRIPT."
+msgstr "Guardar el script de Python presente."
 
-#: mu/interface.py:604
+#: mu/interface/main.py:85
 msgid "Zoom-in"
-msgstr "ZOOM-IN"
+msgstr "Acercar"
 
-#: mu/interface.py:605
+#: mu/interface/main.py:86
 msgid "Zoom in (to make the text bigger)."
-msgstr "ZOOM IN (TO MAKE THE TEXT BIGGER)."
+msgstr "Acercar (para agrandar el texto)."
 
-#: mu/interface.py:606
+#: mu/interface/main.py:87
 msgid "Zoom-out"
-msgstr "ZOOM-OUT"
+msgstr "Alejar"
 
-#: mu/interface.py:607
+#: mu/interface/main.py:88
 msgid "Zoom out (to make the text smaller)."
-msgstr "ZOOM OUT (TO MAKE THE TEXT SMALLER)."
+msgstr "Alejar (para hacer el texto más pequeño)."
 
-#: mu/interface.py:608
+#: mu/interface/main.py:89
 msgid "Theme"
-msgstr "THEME"
+msgstr "Tema"
 
-#: mu/interface.py:609
-msgid "Change theme between day or night."
-msgstr "CHANGE THEME BETWEEN DAY OR NIGHT"
+#: mu/interface/main.py:90
+msgid "Toggle theme between day, night or high contrast."
+msgstr "Alternar tema entre día, noche o alto contraste."
 
-#: mu/interface.py:611
+#: mu/interface/main.py:93
 msgid "Check"
-msgstr "CHECK"
+msgstr "Comprobar"
 
-#: mu/interface.py:612
+#: mu/interface/main.py:94
 msgid "Check your code for mistakes."
-msgstr "CHECK YOUR CODE FOR MISTAKES."
+msgstr "Comprobar código por errores."
 
-#: mu/interface.py:613
+#: mu/interface/main.py:95
 msgid "Help"
-msgstr "HELP"
+msgstr "Ayuda"
 
-#: mu/interface.py:614
+#: mu/interface/main.py:96
 msgid "Show help about Mu in a browser."
-msgstr "SHOW HELP ABOUT MU IN A BROWSER."
+msgstr "Mostrar ayuda de Mu en un navegador."
 
-#: mu/interface.py:615
+#: mu/interface/main.py:97
 msgid "Quit"
-msgstr "QUIT"
+msgstr "Salir"
 
-#: mu/interface.py:616
+#: mu/interface/main.py:98
 msgid "Quit Mu."
-msgstr "QUIT MU."
+msgstr "Salir de Mu."
 
-#: mu/interface.py:695
+#: mu/interface/main.py:177
 msgid "Mu {}"
-msgstr "MU {}"
+msgstr "Mu {}"
```

Comment:
This is for the window title (Mu + version), should this be removed from the translations?